clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -52,36 +56,36 @@ object ClioClient extends LazyLogging {
 
   def main(args: Array[String]): Unit = {
     implicit val system: ActorSystem = ActorSystem(progName)
+    implicit val mat: Materializer = ActorMaterializer()
+
     import system.dispatcher
     sys.addShutdownHook({ val _ = system.terminate() })
 
-    val dispatchFutureOrErr = for {
+    val resultStreamOrErr = for {
       credentials <- AuthUtil
         .getCredentials(
           ClioClientConfig.accessToken,
           ClioClientConfig.serviceAccountJson
         )
         .leftMap(AuthError.apply)
 
-      webClient = new ClioWebClient(
+      webClient = ClioWebClient(
```

Comment:
Could we put a no-arg constructor in ClioWebClient to automagically fill in all these environment parameters?

Reply:
Yeah, I can pull the config-reading into the `apply` method.

Reply:
Er, though that'd make it impossible to inject custom hostname / port / https for integration-testing without generating an application.conf. But I can set these values as default args.